webshare/OWNERS

```diff
@@ -0,0 +1 @@
+@mgiuca
```

Comment:
Commenting on the first file in the directory. Unless https://wicg.github.io/web-share/ is to be renamed, using web-share as the shortname here as well would be good. That is the general pattern, although I can't find it documented.